src/esn.calendar.libs/app/components/event/form/event-form.controller.spec.js

```diff
@@ -357,19 +377,20 @@ describe('The CalEventFormController controller', function() {
         });
 
         calEventServiceMock.createEvent = sinon.spy();
-        this.initController();
-        this.scope.attendees.users[0].freeBusy = 'busy';
-        this.scope.submit();
-        this.rootScope.$digest();
-
-        expect(calEventServiceMock.createEvent).to.not.have.been.called;
-        expect(calEventFreeBusyConfirmationModalService).to.have.been.called;
+        this.initController().then(function() {
+          self.scope.attendees.users[0].freeBusy = 'busy';
+          self.scope.submit();
+          self.rootScope.$digest();
```

Comment:
In the test above, you removed this line, but here you did not. Is there a difference?

Reply:
Nothing, just forgot to remove it :(